app/templates/src/handlebars/helpers/helpers.js

```diff
@@ -11,10 +11,12 @@
  * {{addYear '©'}} --> © 2017
  *
  */
-const addYear = function (s) {
+function addYear(s) {
 	return s + ' ' + new Date().getFullYear();
-};
+}
 
 module.exports = {
-	addYear
+	/* stylelint-disable */ // because handlebars isn't able to work with shorthands
+	addYear: addYear
+	/* stylelint-enable */
```

Comment:
Replace:

```
/* stylelint-disable */ // because handlebars isn't able to work with shorthands
addYear: addYear
* stylelint-enable */
```

with: 

```
addYear: addYear // eslint-disable-next-line object-shorthand
```